09-schema/03-rules.md

```diff
@@ -274,6 +274,112 @@ TypeQLDefine query = TypeQL.define(
 </div>
 
 This rule will make every relation transitive.
+## Efficient transitivity
+
+A common use-case for rules is to infer the transitive closure of a relation. The most straight-forward way of doing this is as follows:
+```typeql
+define
+
+rule transitive-reachability:
+when{
+    (from: $x, to: $y) isa edge;
+    (from: $y, to: $z) isa edge;
+} then {
+    (from: $x, to: $z) isa edge;
+};
+```
+We can interpret this rule as joining two paths together. In a chain `p-q-r-s-t`, to find all nodes reachable from p, we would generate the following relations:
```

Comment:
use "a particular `p`"

Reply:
"we would expect to find the following relations"

Reply:
It's not a particular `p` - it's the `p` in the chain.
If I say "We would expect to find", isn't the reader expecting a contradiction?